test/authentication/test_ed25519.rb

```diff
@@ -24,6 +24,8 @@ def test_no_pwd_key
         signed = priv_key.ssh_do_sign(shared_secret)
         self.assert_equal(true, pub_key.ssh_do_verify(signed, shared_secret))
         self.assert_equal(priv_key.public_key.fingerprint, pub_key.fingerprint)
+        self.assert_equal(priv_key.comment, "vagrant@vagrant-ubuntu-trusty-64")
```

Comment:
IMHO `self` is redundant here. But I added it as the surrounding code was using it.

Should I fix it in the whole file?